README.md

```diff
@@ -152,6 +152,9 @@ Should you need to modify any of the options, you can use the list below to unde
     **NOTE**: Doesn't support overloading currently - all function calls with a matching name will be skipped. 
     
 - `coverageThreshold` - when present Muter will ignore files that have a coverage value less than this option.
+- `indentation` - A object that represent kind and amount of whitespace that should be added when indenting one level. The object value of this property should have exactly one of the following properties:
+    - `spaces` *(number)*: One level of indentation is the given number of spaces.
+    - `tabs` *(number)*: One level of indentation is the given number of tabs.
```

Comment:
For those who aren’t as familiar with the config file syntax, it might be nice to include an example since this is a property (indentation) with a nested sub-property (the number). I’m not sure I’d know how to actually format it.

It would also be nice to describe the default if this value is not provided.

Also:

```suggestion
- `indentation` - A object that represents the kind and amount of whitespace that should be added when indenting one level. The object value of this property should have exactly one of the following properties:
    - `spaces` *(number)*: One level of indentation is the given number of spaces.
    - `tabs` *(number)*: One level of indentation is the given number of tabs.
```

Reply:
Thanks for reminding me. I've updated all the examples project's muter configuration